src/multiversioning.zig

```diff
@@ -365,11 +372,14 @@ pub const MultiversionHeader = extern struct {
     past: PastReleases = .{},
     past_padding: [16]u8 = std.mem.zeroes([16]u8),
 
+    current_git_commit: [20]u8,
```

Comment:
I'm going to claim I left these out on purpose, to test adding new fields :smile:.